WCON_format.md

```diff
@@ -355,8 +356,7 @@ Here is an example WCON file using origin and centroid points:
     "units":{"t":"s", "x":"mm", "y":"mm"},
     "data":[
  	      { "id":1, "t":1.3, "x":[7.2, 8.1], "y":[0.5, 0.3],
-          "ox":32.4, "oy":9.2, "cx":7.676, "cy": 0.384
-        }
+          "ox":32.4, "oy":9.2, "@SmithLab": {"cx":7.676, "cy": 0.384}}
```

Comment:
You're right - it looks like I somehow missed implementing centroid.  I only implemented origin ("ox", "oy").  I'll add that now.